src/Core/Domain/Product/Command/UpdateProductSeoCommand.php

```diff
@@ -61,6 +61,8 @@ class UpdateProductSeoCommand
      */
     private $redirectOption;
 
+    //@todo; where is $product->indexed prop?
```

Comment:
🤔  ? I dont understand

Reply:
reminder that we didn't yet handle $product->indexed in any command of product migration. So I wonder if we didn't miss anything